sender.toml.example

```diff
@@ -3,10 +3,10 @@
 amqp_url = "amqp://localhost/%2Ffmn"
 queue = "email"
 
-[tls]
-ca_cert = "/etc/fedora-messaging/cacert.pem"
-keyfile = "/etc/fedora-messaging/fmn-key.pem"
-certfile = "/etc/fedora-messaging/fmn-cert.pem"
+# [tls]
+# ca_cert = "/etc/fedora-messaging/cacert.pem"
+# keyfile = "/etc/fedora-messaging/sender-key.pem"
+# certfile = "/etc/fedora-messaging/sender-cert.pem"
```

Comment:
Same question as elsewhere, don’t we need SSL/TLS configuration any longer? And is the renaming from `fmn-*.pem` to `sender-*.pem` intentional?

Reply:
Yeah we need to be able to do both: in prod it will be a connection to our main broker, and it requires SSL. For development we connect to rabbitmq on tinystage in plain text.